virttest/utils_libvirt/libvirt_audit.py

```diff
@@ -0,0 +1,38 @@
+"""
+libvirt audit related utility functions
```

Comment:
Is libvirt audit very significant feature to be wrapped up in a single file?

Reply:
@dzhengfy "Is libvirt audit very significant feature to be wrapped up in a single file?" I wonder why you believe only  very significant feature  can be wrapped in single file ? and How you evaluate its significance?

In terms of audit  libvirt feature world, if you take a look at those cases against them in polarion, they are clearly separated with others

Reply:
@dzhengfy "Is libvirt audit very significant feature to be wrapped up in a single file?" I wonder why you believe only very significant feature can be wrapped in single file ? and How you evaluate its significance?

In terms of audit libvirt feature world, if you take a look at those cases against them in polarion, they are clearly separated with others